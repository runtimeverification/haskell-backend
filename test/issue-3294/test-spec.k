module TEST-SPEC
  imports TEST

  claim <k> intersect ( intersectSet( SetItem(_X) SetItem(a), SetItem(b))) => .K </k>
endmodule
