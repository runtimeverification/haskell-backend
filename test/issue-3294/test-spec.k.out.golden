  #Not ( {
    _X
  #Equals
    a
  } )
#And
  <k>
    result ( intersectSet ( SetItem ( _X )
    SetItem ( a ) , SetItem ( b ) ) ) ~> .
  </k>
